python/tvm/micro/base.py

```diff
@@ -50,12 +50,18 @@ class Session:
     .. code-block:: python
 
       c_mod = ...  # some module generated with "c" as the target
-      device_type = "host"
-      with tvm.micro.Session(device_type) as sess:
-          sess.create_micro_mod(c_mod)
+      device_type = "openocd"
+      toolchain_prefix = "riscv64-unknown-elf-"
+      with tvm.micro.Session(device_type,
+                             toolchain_prefix,
+                             base_addr=0x10010000,
```

Comment:
can we have a good base_addr default?

Reply:
I'm not sure what a good default would be, since it will be very device-dependent.  That's why I'm leaving it at zero for now.

Another option would be to make it a required arg, but then that's weird for using the host device, since the host sets its own base address (wherever the mmapped region is).

Reply:
OK, let us default to None, allow default for devices we know and raise error when it is needed(but has not been passed in)